guides/common/modules/proc_resetting-external-authentication-configuration-for-kerberos-sso.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="resetting-external-authentication-configuration-for-kerberos-sso"]
+= Resetting external authentication configuration for Kerberos SSO
+
+[role="_abstract"]
+You can disable external authentication with {FreeIPA} by resetting the external authentication configuration.
+This prevents user accounts defined in the external authentication source from accessing {Project}.
+
+[IMPORTANT]
+====
+Resetting external authentication prevents users from accessing {Project} as described in xref:configuring-kerberos-sso-with-{FreeIPA-context}-in-{project-context}[] and xref:configuring-kerberos-sso-for-active-directory-users-in-project_{context}[].
+However, some configuration files, such as configuration files for the System Security Services Daemon (SSSD), will remain modified because {Project} does not have access to the previous state of these files.
+====
+
+.Procedure
+ifdef::foremanctl[]
+* Reset {FreeIPA} authentication configuration to the default state:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foremanctl deploy --reset-external-authentication
+----
+endif::[]
+ifndef::foremanctl[]
+* Reset {FreeIPA} authentication configuration to the default state:
+** To disable access to {ProjectWebUI}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --reset-foreman-ipa-authentication
+----
+** To disable access to {ProjectWebUI}, {Project} API, and Hammer CLI:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --reset-foreman-ipa-authentication-api
+----
+endif::[]
+
+.Verification
+ifdef::foremanctl[]
+#How can users verify this?#
+endif::[]
```

Comment:
Not sure if output of foremanctl is stable enough to do something like we do with the old installer. The following is a bit of a poor man's solution, but it should work.

```
$ curl -k -u : --negotiate https://foreman.example.com/users/extlogin
<html><body>You are being <a href="https://foreman.example.com/users/login">redirected</a>.</body></html>
```

Reply:
> Not sure if output of foremanctl is stable enough to do something like we do with the old installer. The following is a bit of a poor man's solution, but it should work.
> 
> ```
> $ curl -k -u : --negotiate https://foreman.example.com/users/extlogin
> <html><body>You are being <a href="https://foreman.example.com/users/login">redirected</a>.</body></html>
> ```

I was trying to find out what the message looks like when external authentication is not configured (because this is a section on _resetting_ the ext auth source). However, I still got the `You are being redirected` message. This is on a server that has never had ext auth configured.

So now I wonder what I'm doing wrong. What should the message look like on a system where ext auth has been reset?